envs/api-python-s3-k8s/.whirl.env

```diff
@@ -0,0 +1,44 @@
+AWS_ACCESS_KEY_ID=bar
+AWS_SECRET_ACCESS_KEY=foo
+DEMO_BUCKET=demo-s3-output
+S3_LOG_BUCKET=airflow-s3-logs
+AWS_SERVER=s3server
+AWS_PORT=4563
+PORT_WEB_UI=8080
+
+# postgres env vars
+POSTGRES_HOST=postgresdb
+POSTGRES_PORT=5432
+POSTGRES_PASSWORD=pAssw0rd
+POSTGRES_USER=airflow
+POSTGRES_DB=airflow
+
+# Airflow env vars
+AIRFLOW__CORE__EXPOSE_CONFIG=True
+AIRFLOW__WEBSERVER__EXPOSE_CONFIG=True
+AIRFLOW__CORE__LOAD_DEFAULT_CONNECTIONS=False
+AIRFLOW__CORE__LOAD_EXAMPLES=False
+AIRFLOW__CORE__FERNET_KEY=W5gmA+dp84hkZEzpxPw4LTmhbXA1uVxKZsgIfay8wno=
```

Comment:
bit annoying that we have to set these same values in multiple places...  But I don't really see an easy/feasible alternative tbh (jinja rendering is a bit overkill...)